src/mito2/src/read/dedup.rs

```diff
@@ -502,6 +502,9 @@ pub(crate) struct LastNonNullIter<I> {
     /// fetch a new batch.
     /// The batch is always not empty.
     current_batch: Option<Batch>,
+    // The index of the current row in the current batch.
+    // more to check issue #5229.
```

Comment:
nit: 

```suggestion
    /// The index of the current row in the current batch.
    // more to check issue #5229.
```

(rust doc comment v.s. normal comment)